api/v1/services/UserService.js

```diff
@@ -118,3 +118,13 @@ module.exports.resetPassword = function (user, password) {
 			return updated.save();
 		});
 };
+
+module.exports.deleteUser = function (user) {
+	if(user.hasRole(utils.roles.ATTENDEE, false)) {
```

Comment:
This is a bit too specific. If it can delete any user, then it should check if they have any role at all, not just the attendee one.